pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Mistake Odyssey"
+Subtitle: "Git, Github, VS Code and Markdown for non-programmers"
+date: 2021-07-08T06:00:00+01:00
```

Comment:
Could you change the date to `07-15`? I think that we should merge blog post tomorrow :)

```suggestion
date: 2021-07-15T06:00:00+01:00
```